src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1513,6 +1513,21 @@ public bool IsDeclaration(SyntaxNode node)
             return SyntaxFacts.IsNamespaceMemberDeclaration(node.Kind()) || IsMemberDeclaration(node);
         }
 
+        public bool IsTypeDeclaration(SyntaxNode node)
```

Comment:
Is this just IsNamespaceMemberDeclaration except for the namespace case? Can we share code?